Makefile

```diff
@@ -49,6 +49,7 @@ check: flake8 test
 .PHONY: pylint
 pylint:
 	pylint -d missing-docstring *.py kiwi_lint/
+	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=tcms.settings.test pylint --load-plugins=pylint_django.checkers.migrations -d missing-docstring -d duplicate-code  tcms/*/migrations/*
```

Comment:
Use the `$(DJANGO_SETTINGS_MODULE)` variable here. Don't hard-code the settings and don't copy paste